sandboxes/dynamicanalysis/analyze-node.js

```diff
@@ -1,23 +1,27 @@
 #!/usr/bin/env node
 
-const { spawnSync } = require('child_process');
-const { argv } = require('process');
+// eslint no-var: 0
+// jshint esversion: 6
+// jshint node: true
+'use strict';
 
-function install(package) {
+const {spawnSync} = require('child_process');
+const fs = require('fs');
+const process = require('process');
+
+const executionLogPath = '/execution.log';
```

Comment:
So this will be created by the package analysis worker before calling this right? And that's where we add the control to disable this for node for now?

Reply:
yes that's right